jsonschema/validators.py

```diff
@@ -114,13 +120,15 @@ def _store_schema_list():
 
 
 def create(
-    meta_schema,
-    validators=(),
+    meta_schema: _typing.Schema,
```

Comment:
I have to double check (which I will be lazy and not do here) but Unlike `_typing.Schema`, meta-schemas (in the library, but possibly also according to the spec, I forget) *must* be objects I think, not `bool | thing`, so you'll need two types I suspect?

Reply:
Okay, that makes sense. I need to think harder about what to do with schemas more broadly, but we now have a clean place where we can define `_typing.MetaSchema`. I'll make that tweak.